libraries/stdlib/test/collections/CollectionTest.kt

```diff
@@ -854,4 +854,41 @@ class CollectionTest {
         // we need toString() inside pattern because of KT-8666
         assertEquals("[1, a, null, ${Long.MAX_VALUE.toString()}]", listOf(1, "a", null, Long.MAX_VALUE).toString())
     }
+
+    @test fun testGetRange() {
+        val listStr = listOf("a", "b", "c", "d", "e")
```

Comment:
I believe we also need tests on an empty list: should fail at any range but empty range on empty list should produce empty list